src/libpriv/rpmostree-postprocess.cxx

```diff
@@ -387,11 +387,21 @@ postprocess_final (int rootfs_dfd, rpmostreecxx::Treefile &treefile, gboolean un
     {
       g_print ("Recompiling policy\n");
 
-      /* Now regenerate SELinux policy so that postprocess scripts from users and from us
-       * (e.g. the /etc/default/useradd incision) that affect it are baked in. */
-      rust::Vec child_argv = { rust::String ("semodule"), rust::String ("-nB") };
-      ROSCXX_TRY (bubblewrap_run_sync (rootfs_dfd, child_argv, false, (bool)unified_core_mode),
-                  error);
+      {
+        /* Now regenerate SELinux policy so that postprocess scripts from users and from us
+         * (e.g. the /etc/default/useradd incision) that affect it are baked in. */
+        rust::Vec child_argv = { rust::String ("semodule"), rust::String ("-nB") };
+        ROSCXX_TRY (bubblewrap_run_sync (rootfs_dfd, child_argv, false, (bool)unified_core_mode),
+                    error);
+      }
+
+      /* Temporary workaround for https://github.com/openshift/os/issues/1036. */
+      {
+        rust::Vec child_argv = { rust::String ("semodule"), rust::String ("-n"),
+                                 rust::String ("--rebuild-if-modules-changed") };
```

Comment:
The reason I went with `--rebuild-if-modules-changed` here instead of the newer `--refresh` is that the latter is not in f36 and while f36 isn't affected by this bug, I'd like this patch in place ahead of time.

Reply:
Also, I didn't add a feature check here for `--rebuild-if-modules-changed`. I *think* all the userspaces we care about for the `main` branch have that patch now.

@cgwalters I think we'll need to backport this to the `rhel8` branch, right?

Reply:
Oh blah.  So we have newer semodule in rhel8 versus f36?? :cry: 

> @cgwalters I think we'll need to backport this to the rhel8 branch, right?

Yeah